lldb/test/API/lang/swift/async/expr/TestSwiftAsyncExpressions.py

```diff
@@ -16,6 +16,20 @@ class TestSwiftAsyncExpressions(lldbtest.TestBase):
     def test_actor(self):
         """Test async unwind"""
         self.build()
+        self.do_test_actor()
+
+    @swiftTest
+    @skipIfWindows
+    @skipIfLinux
+    @skipIf(archs=no_match(["arm64", "arm64e", "arm64_32", "x86_64"]))
```

Comment:
Why is this necessary? It sounds like this basically excludes i386-apple-macos11, which I think isn't a thing?
On the other hand we may need to `@skipDarwinEmbedded` since the triple explictly asks for macos.

Reply:
I'll review and fix these. You're no doubt right. I duplicated the existing function and kept its decorations without looking closely at them.